tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+envlist = flake8,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          pypy,
+          pypy3,
+          coverage
+
+[testenv]
+usedevelop = true
+commands =
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:coverage]
+basepython = python3.7
```

Comment:
The base python in those places where you want to force it (such as the coverage report and linting) should be identical.

Reply:
Actually, I'm in favor of linting with each supported version of Python, rather than just one, so only assembling the coverage report should run with a single version.

Reply:
For a second there I thought I saw python2.7 and was about to object ;)

Reply:
As the near future Python 2 support can/will be dropped I see no longer a gain in running coverage for multiple Python versions.

Reply:
IMHO linting isn't important enough to sit through the process for each supported Python version. Just select one, fix the basepython setting to it, and then make the naming generic. No `lint-py36` anymore like we have for Zope, I always hated that name.